cmd/shield/help/update-target

```diff
@@ -32,7 +32,7 @@ USAGE: @G{shield} update-target --tenant @Y{TENANT} [OPTIONS] @Y{NAME-OR-UUID}
 
   -C, --compression
                     The type of compression to use when backing up this
-                    target. Valid values are `bzip2' and `none'. Defaults
+                    target. Valid values are `bzip2' 'gzip' and `none'. Defaults
```

Comment:
Formatting is inconsistent on this line, should have commas ',' between options.